### PR TITLE
Adds sounds for clapping, adds a golfclap emote.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Adds a sound to the *clap emote.

Adds a new emote, *golfclap, essentially clapping but just disappointed clapping. Uses another soundbyte.

Changelog SOON

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Comment by NonQueueingMatt** [comment]
why travis gotta be a bitch

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -111,9 +111,19 @@
...
 			if (!src.restrained())
 				message = "claps."
+				playsound(loc, 'sound/effects/clap.ogg', 50, 1)
 				m_type = 2
 				if(miming)
 					m_type = 1
+
+		if ("golfclap")
+			if (!src.restrained())
+				message "claps, clearly unimpressed."
```
Comment (BurgerLUA) [inline code/modules/mob/living/carbon/human/emote.dm]: should be ``message = "claps, clearly unimpressed."

**Comment by NonQueueingMatt** [comment]
it is the big fixed

*medic

**Comment by NonQueueingMatt** [comment]
i will destroy travis if it keeps failing

**Comment by BurgerLUA** [comment]
get a dev to restart travis, travis failed because of a bug not because of you

**Pull request merged**